### DISCUSSION TITLE
Which is the fastest model to train for voice cloning?

### DISCUSSION THREAD

**Discussion by tuapuikia** [coqui-ai/TTS#2421] [General Q&A] [answered]
Hi,

I tried using Glow-TTS but so far it hasn't worked, as it only produces noise after hours of training. My sample is a single line with one sample.

Is it possible to train with a lossless model for quicker results? I'm using the recipe from 
https://github.com/coqui-ai/TTS/blob/dev/recipes/ljspeech/glow_tts/train_glowtts.py.

Regards,
Pat

**Comment by erogol** [comment, marked as answer]
speedy speech is the fastest.

**Reply by tuapuikia** [reply]
Thank you for the reply, I will try it.
